### PR TITLE
add master outputs and rename current ones

### PR DESCRIPTION
**Pull request by fgimenez**
Towards https://github.com/giantswarm/giantswarm/issues/2023

Introduced new fields for master's imageID and CloudConfig version, also removed ClusterVersion and added WorkerCloudConfig version and renamed worker-related fields. I've taken the opportunity to promote master and worker image ID from `launch_config` and `instance` to the main adapter.

Still not doing phased updates, the complete stack is updated as a whole.

### CODE REVIEW

File: service/templates/cloudformation/guest/outputs.yaml [hunk 1]
```diff
@@ -1,9 +1,13 @@
 {{define "outputs"}}
 Outputs:
-  WorkersOutput:
+  MasterImageID:
+    Value: {{ .MasterImageID }}
+  MasterCloudConfigVersion:
+    Value: {{ .MasterCloudConfigVersion }}
+  Workers:
```
Comment (rossf7) [inline service/templates/cloudformation/guest/outputs.yaml]: I think WorkerCount would be a clearer name?

File: service/templates/cloudformation/guest/outputs.yaml [hunk 2]
```diff
@@ -1,9 +1,13 @@
 {{define "outputs"}}
 Outputs:
-  WorkersOutput:
+  MasterImageID:
+    Value: {{ .MasterImageID }}
```
Comment (rossf7) [inline service/templates/cloudformation/guest/outputs.yaml]: I think we should add outputs for master and worker instance types.

We don't support changing instance types yet but it should be straight forward to implement later. WDYT?

File: service/templates/cloudformation/guest/outputs.yaml (hunk 1 above)
Comment (fgimenez) [reply service/templates/cloudformation/guest/outputs.yaml]: Much better, done.

File: service/templates/cloudformation/guest/outputs.yaml (hunk 2 above)
Comment (fgimenez) [reply service/templates/cloudformation/guest/outputs.yaml]: Totally, already added, PTAL.

**Pull request merged**